### ISSUE TITLE
Why logs during training doesn't show validation loss?

### ISSUE THREAD

**Issue by CaliCuty** [ultralytics/ultralytics#20163]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I can see train box_loss, cls_loss and dfl_loss per batch during training. However, during validation phase after each epoch, why logs doesn' show loss but showing box P/R/mAP50/mAP50-95 only. Is there a way to monitor validation loss during training?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @CaliCuty, thank you for reaching out and for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com/) to explore a variety of [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. Many common questions are addressed there.

Regarding your inquiry, if this is a ❓ Question, please provide as much detail as possible, such as specific logs, configurations, or any additional context that might help us understand your concern better. This ensures we can assist you more effectively.

If you believe this might be a 🐛 Bug Report, it'd be helpful to share a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help debug the issue.

Stay connected with the Ultralytics community! For real-time discussions, join us on [Discord](https://discord.com/invite/ultralytics) 🎧. For in-depth discussions, visit [Discourse](https://community.ultralytics.com/), or engage with fellow users on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

Make sure you're using the latest version of the `ultralytics` package. Upgrade in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) by running:

```bash
pip install -U ultralytics
```

This ensures you have the latest features and fixes, which might already address your issue.

## Environments

YOLO can be run in verified environments with all dependencies preinstalled, including:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM ([GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon** Deep Learning AMI ([AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker Image** ([Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. These tests verify the correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu.

This is an automated response to assist you quickly. An Ultralytics engineer will review your issue and provide further assistance soon. Thanks for your patience! 😊

**Comment by Y-T-G** [comment]
You can view it on tensorboard.
